### PR TITLE
Implement Exercise: Acronym

### PR DESCRIPTION
**Pull request by Dwivedirahul44**
Ref: https://github.com/exercism/abap/issues/221

### CODE REVIEW

**Comment by Dwivedirahul44** [comment]
./exercises/practice/acronym/zcl_acronym.clas.testclasses.abap[7, 10] - Variable "CUT" contains unknown: REF, unable to resolve zcl_acronym (unknown_types) [E]
./exercises/practice/acronym/zcl_acronym.clas.testclasses.abap[30, 5] - Method "parse" not found, methodCallChain (check_syntax) [E]

I dont understand why its failing on this.

File: exercises/practice/acronym/.meta/zcl_acronym_example.clas.abap [hunk 1]
```diff
@@ -0,0 +1,29 @@
+CLASS zcl_acronym_example DEFINITION
```
Comment (larshp) [inline exercises/practice/acronym/.meta/zcl_acronym_example.clas.abap:1]: try renaming this class to `zcl_acronym`

during CI it is checked that this solution works with the unit tests

Comment (Dwivedirahul44) [reply exercises/practice/acronym/.meta/zcl_acronym_example.clas.abap]: thanks, i renamed it.

**Comment by larshp** [comment]
🎉

File: exercises/practice/acronym/zcl_acronym.clas.testclasses.abap [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    METHODS setup.
+    METHODS:
+      basic_phrase FOR TESTING,
+      lowercase_words FOR TESTING,
+      punctuations FOR TESTING,
+      all_caps FOR TESTING,
+      punc_wo_space FOR TESTING,
+      long_phrase FOR TESTING,
+      consecutive_delimeters FOR TESTING ,
```
Comment (larshp) [inline exercises/practice/acronym/zcl_acronym.clas.testclasses.abap:17]:
```suggestion
      consecutive_delimeters FOR TESTING,
```

File: exercises/practice/acronym/zcl_acronym.clas.testclasses.abap [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+
+CLASS ltcl_acronym  IMPLEMENTATION.
+  METHOD setup.
+    cut = NEW zcl_acronym( ).
+  ENDMETHOD.
+
+  METHOD basic_phrase.
+    cl_abap_unit_assert=>assert_equals(
+       act = cut->parse( `Portable Network Graphics`)
```
Comment (larshp) [inline exercises/practice/acronym/zcl_acronym.clas.testclasses.abap:31]: please align the use of whitespace

File: exercises/practice/acronym/zcl_acronym.clas.testclasses.abap (hunk 2 above)
Comment (Dwivedirahul44) [reply exercises/practice/acronym/zcl_acronym.clas.testclasses.abap]: thanks, changed.

File: exercises/practice/acronym/zcl_acronym.clas.testclasses.abap (hunk 3 above)
Comment (Dwivedirahul44) [reply exercises/practice/acronym/zcl_acronym.clas.testclasses.abap]: thanks, made changes to all the calls and aligned the whitespaces.

**Review by larshp — APPROVED** [review]

**Comment by larshp** [comment]
bugfix = https://github.com/abaplint/abaplint/pull/2735 to check the missing whitespace

**Comment by Dwivedirahul44** [comment]
closed by mistake, opened again. Thanks for the pointers, please do the needful

**Comment by larshp** [comment]
merging, thanks 🚢

**Pull request merged**